src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotCorrectedObject = {};
+  const swappedValues = {};
+
+  for (const key in robot) {
+    if (robot.hasOwnProperty(key)) {
+      const value = robot[key];
+
+      if (swappedValues.hasOwnProperty(value)) {
+        return null;
+      }
+
+      robotCorrectedObject[value] = key;
```

Comment:
Instead of using 'swappedValues[value] = true;', update the 'invertedRobot' directly, i.e., 'invertedRobot[value] = key;'.